placeholder_images.py

```diff
@@ -0,0 +1,65 @@
+from wmfdata.spark import get_session
+import argparse
+import requests
+import pandas as pd
+import numpy as np
+import findspark
+
+findspark.init('/usr/lib/spark2')
+
+spark = get_session(type='regular', app_name="ImageRec-DEV Training")
```

Comment:
Small nit: for spark jobs submitted outside a notebook I'd prefer if we'd use plain config files instead of `wmfdata`. The library fantastic for notebook environments, but the cli offer some more flexibility (and a configuration that we can version control and share across jobs / languages). 

There's an example in `transform.py`. SparkSession is initialised as
```
spark = SparkSession.builder.getOrCreate()
```

And when you launch the job with `spark-submit2` a config can be specified with
```
spark2-submit --properties-file ${spark_config}
```

One thing I like about this approach is that it let us decouple the runtime from the job's code. With `wmfdata`, every time we run the script we assume it will run on the Analytics Cluster (`type='regular'`). With a config based approach it's easy to deploy to other targets, without altering the script.